src/ng-jhipster.module.ts

```diff
@@ -43,55 +38,16 @@ export function missingTranslationHandler(configService: JhiConfigService) {
 }
 
 @NgModule({
-    imports: [
-        CommonModule,
-        TranslateModule.forRoot({
-            loader: {
-                provide: TranslateLoader,
-                useFactory: translatePartialLoader,
-                deps: [HttpClient]
-            },
-            missingTranslationHandler: {
-                provide: MissingTranslationHandler,
-                useFactory: missingTranslationHandler,
-                deps: [JhiConfigService]
-            }
-        }),
-        CommonModule,
-        NgbModule,
-        FormsModule
-    ],
+    imports: [CommonModule, NgbModule, FormsModule],
     declarations: [...JHI_PIPES, ...JHI_DIRECTIVES, ...JHI_COMPONENTS, JhiTranslateDirective],
     entryComponents: [JhiThreadModalComponent],
-    exports: [...JHI_PIPES, ...JHI_DIRECTIVES, ...JHI_COMPONENTS, JhiTranslateDirective, TranslateModule, CommonModule]
+    exports: [...JHI_PIPES, ...JHI_DIRECTIVES, ...JHI_COMPONENTS, JhiTranslateDirective, CommonModule]
 })
 export class NgJhipsterModule {
     static forRoot(moduleConfig: JhiModuleConfig): ModuleWithProviders {
```

Comment:
I think both `forRoot` and `forChild` expose same services via factory method, do we need `forChild` ?

Reply:
forChild is redundant, I removed this with new commit